subsys/ipc/ipc_service/backends/ipc_rpmsg_multi_core.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2021, Carlo Caione <ccaione@baylibre.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ipc/ipc_service_backend.h>
+#include <ipc/rpmsg_multi_instance.h>
+
+#include <logging/log.h>
+#include <sys/util_macro.h>
+#include <zephyr.h>
+#include <device.h>
+
+#define DT_DRV_COMPAT ipc_rpmsg_mc
```

Comment:
Name of this macro gets a little confusing when we start adding system configuration alongside hardware description to devicetree. I think it could be renamed to `DT_NODE_COMPAT`, or an alias could be created. But it's out of scope of this patch.

Reply:
I agree it is odd to see DT_DRV_COMPAT being used in subsys/ instead of drivers/. That's a code smell for sure, though maybe there's a good reason for it; I'm just digging into this.